app/src/androidTest/java/org/mozilla/fenix/ui/SettingsBasicsTest.kt

```diff
@@ -120,6 +120,7 @@ class SettingsBasicsTest {
 
         navigationToolbar {
         }.enterURLAndEnterToBrowser(page2.url) {
+            verifyUrl(page2.url.toString())
```

Comment:
From the test recording it looks like the main menu is opened before the browser manages to completely load the second webpage, so the bookmark button is not in the correct state.
Added this step to make sure the page is on the right URL before proceeding to save it as bookmark